<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbManagerDlgImportVector</class>
 <widget class="QDialog" name="DbManagerDlgImportVector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import vector layer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wdgInput" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="btnChooseInputFile">
        <property name="text">
         <string>â€¦</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cboInputLayer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="chkSelectedFeatures">
          <property name="text">
           <string>Import only selected features</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Output table</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Schema</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboSchema">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Table</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>cboTable</zorder>
     <zorder>label_2</zorder>
     <zorder>cboSchema</zorder>
     <zorder>label_3</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="chkDropTable">
        <property name="text">
         <string>Replace destination table (if exists)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkPrimaryKey">
        <property name="text">
         <string>Primary key</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editPrimaryKey">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkGeomColumn">
        <property name="text">
         <string>Geometry column</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editGeomColumn">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="chkEncoding">
        <property name="toolTip">
         <string>Input layer encoding</string>
        </property>
        <property name="text">
         <string>Encoding</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cboEncoding">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QCheckBox" name="chkSinglePart">
        <property name="text">
         <string>Do not promote to multi-part</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QCheckBox" name="chkSpatialIndex">
        <property name="text">
         <string>Create spatial index</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QCheckBox" name="chkLowercaseFieldNames">
        <property name="text">
         <string>Convert field names to lowercase</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkTargetSrid">
        <property name="text">
         <string>Target SRID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QgsProjectionSelectionWidget" name="widgetTargetSrid" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsProjectionSelectionWidget" name="widgetSourceSrid" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkSourceSrid">
        <property name="text">
         <string>Source SRID</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="chkCom">
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QTextEdit" name="editCom">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>groupBox_2</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>wdgInput</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cboInputLayer</tabstop>
  <tabstop>btnChooseInputFile</tabstop>
  <tabstop>chkSelectedFeatures</tabstop>
  <tabstop>cboSchema</tabstop>
  <tabstop>cboTable</tabstop>
  <tabstop>chkPrimaryKey</tabstop>
  <tabstop>editPrimaryKey</tabstop>
  <tabstop>chkGeomColumn</tabstop>
  <tabstop>editGeomColumn</tabstop>
  <tabstop>chkSourceSrid</tabstop>
  <tabstop>chkTargetSrid</tabstop>
  <tabstop>chkEncoding</tabstop>
  <tabstop>cboEncoding</tabstop>
  <tabstop>chkDropTable</tabstop>
  <tabstop>chkSinglePart</tabstop>
  <tabstop>chkLowercaseFieldNames</tabstop>
  <tabstop>chkSpatialIndex</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbManagerDlgImportVector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPrimaryKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>editPrimaryKey</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkGeomColumn</sender>
   <signal>toggled(bool)</signal>
   <receiver>editGeomColumn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEncoding</sender>
   <signal>toggled(bool)</signal>
   <receiver>cboEncoding</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbManagerDlgImportVector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSourceSrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>widgetSourceSrid</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTargetSrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>widgetTargetSrid</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkCom</sender>
   <signal>toggled(bool)</signal>
   <receiver>editCom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>553</y>

    </hint>
   </hints>
  </connection>
 </connections>
</ui>
